src/Sentry/Platforms/Android/AndroidEventProcessor.cs

```diff
@@ -9,17 +9,17 @@ internal class AndroidEventProcessor : ISentryEventProcessor, IDisposable
     private readonly JavaSdk.IEventProcessor? _androidProcessor;
     private readonly JavaSdk.Hint _hint = new();
 
-    public AndroidEventProcessor(SentryAndroidOptions androidOptions)
+    public AndroidEventProcessor(SentryAndroidOptions nativeOptions)
```

Comment:
Should class `SentryAndroidOptions` also become `SentryNativeOptions` or that's rather confusing because `sentry-native`?

Although I see it being renamed below, did we have 2 `SentryAndroidOptions`s?